manifests/alert_cycle.pp

```diff
@@ -0,0 +1,40 @@
+# @summary Pauses and starts (cycles) alerts on a schedule
+#
+# @param crons
+#   Hash of CRON entries for pausing/starting alerts.
+#   For the command key, the path to a alert_toggle script should be omitted
+#
+# @param cycle_alerts
+#   Enable or disable the cycling of alerts
+#
+# @param script_path
+#   Path to the alert_toggle.sh script
+#   
+class profile_metrics_alerting::alert_cycle (
+  Hash $crons,
+  Boolean $cycle_alerts,
+  String $script_path,
+) {
+
+  Cron {
+    user => 'root',
+  }
+
+  if ($cycle_alerts) {
+    $crons.each | $k, $v | {
```

Comment:
This is nice code. I want to steal this for other profiles.

Reply:
It's mostly stolen from here: https://github.com/ncsa/puppet-profile_audit/blob/main/manifests/root_equivalence.pp

I just added some lines to have the crons get deleted when not needed, and some lines to customize the command given so I don't need to give it the script path each time.